VirtoCommerce.CatalogModule.Data/packages.config

```diff
@@ -5,7 +5,7 @@
   <package id="Common.Logging.Core" version="3.3.1" targetFramework="net461" />
   <package id="EntityFramework" version="6.2.0" targetFramework="net461" />
   <package id="FluentValidation" version="7.5.2" targetFramework="net461" />
-  <package id="Hangfire.Core" version="1.6.17" targetFramework="net461" />
+  <package id="Hangfire.Core" version="1.6.19" targetFramework="net461" />
```

Comment:
Why do we need to update it?
Could you revert this change?

Reply:
reverted